Dockerfile

```diff
@@ -10,9 +10,6 @@ COPY preparermd.gemspec /usr/src/app/preparermd.gemspec
 COPY lib/preparermd/version.rb /usr/src/app/lib/preparermd/version.rb
 RUN bundle install
 
-COPY . /usr/src/app
```

Comment:
Did you mean to get rid of this `COPY`? It's what brings most of the source into the container, when you're not running locally with `/usr/src/app` volume-mounted.